Merge non-reviewed changes into development (#361)
Lembas-Modding-Team/pvp-mode: src/main/resources/assets/pvpmode/modules/lotr/default_safe_biomes.txt

# This file lists all biome and alignment dependent overrides, forcing PvP mode to OFF when conditions are met.
#
# Basically this file allows you to specify in which biome a player which is aligned to a faction you assign to this biome is forced to have PvP disabled.

> # Also, you can define what "aligned" means (in terms of alignment and pledging): If the alignment of the player is greater than the specified one for the pledging state of the player, the player is seen as aligned.

Author: ```suggestion
# Also, you can define what "aligned" means (in terms of alignment and pledging): if the alignment of the player is greater than the one specified for the pledging state of the player, the player is seen as aligned.
```

> # Also, you can define what "aligned" means (in terms of alignment and pledging): If the alignment of the player is greater than the specified one, the player is seen as aligned, pledging can also be taken into account.

Author: ```suggestion
# Also, you can define what "aligned" means (in terms of alignment and pledging): If the alignment of the player is greater than the specified one for the pledging state of the player, the player is seen as aligned.
```

# The first part is the faction name, which is the name which is internally used by the game (see the command /alignment). Also you can use the specifier ALL,
# which means that every player, regardless of his alignment, is forced to have PvP disabled on the specified biomes.
#
# The second part is the alignment specifier. It's defined with minAlignmentNoPledge/minAlignmentWithPledge. 

> # "minAlignmentNoPledge" is the minimum required alignment if the player didn't pledge to the specified faction, "minAlignmentWithPledge" is the minimum required alignment if the player did pledge to the specified faction.

Author: ```suggestion
# "minAlignmentNoPledge" is the minimum required alignment if the player is not pledged to the specified faction, "minAlignmentWithPledge" is the minimum required alignment if the player is pledged to the specified faction.
```

> # If you only want to specify the alignment value without pledging, just write the number, without anything else. Whether the player did pledge or not is not important then.

Author: ```suggestion
# If you only want to specify the alignment value without pledging, just write the number, without anything else. In this case, it is not taken into account whether the player is pledged to the specified faction or not.
```

> # Write "/minAlignmentWithPledge" to specify only the alignment value with pledging. This means that the player has to pledge to the specified faction.

Author: ```suggestion
# Write "/minAlignmentWithPledge" to only specify the alignment value with pledging. This means that the player has to be pledged to the specified faction.
```

> # If the specified faction is ALL, the alignment specifier will be ignored (but still has to be a valid one).

Author: ```suggestion
# If the specified faction is ALL, the alignment specifier will be ignored. (But it still has to be valid!)
```
